src/lua/minifio.c

```diff
@@ -53,6 +53,12 @@ tarantool_lua_minifio_init(struct lua_State *L)
 		{NULL, NULL}
 	};
 
+	/*
+	 * The loaders.builtin loader is not in effect yet.
+	 * Set the minifio module into package.loaded manually.
+	 */
+	lua_getfield(L, LUA_REGISTRYINDEX, "_LOADED");
```

Comment:
Minor: why do we need use `_LOADED` here instead of `_PRELOAD`?

Reply:
I have several answers here.

1. Why should we use `_PRELOAD` if we need only `_LOADED`? If we would use `_PRELOAD` here, the result would be setting the value into `_LOADED` in a bit less direct way.
2. If there would be the builtin loader at this stage, `require()` would find the module and store it in `_LOADED`. We imitate this behavior.
3. `_PRELOAD` is accessible by a user and I prefer to don't touch it in the platform if possible.

Reply:
1. `package.loaded[module] = nil` is a standard idiom for module "unload"; in general case `package.preload` allows to reload the module if this is needed.
2. Yeah, but see the reason above.
3. Don't get this point. `package.loaded` is accessible too, isn't it?

Reply:
We agreed that we can't reach an agreement here, so I don't know. I'll leave it as is.